src/main/java/org/reactome/release/qa/check/ChimericInstancesCheck.java

```diff
@@ -32,6 +32,7 @@
 @SliceQACheck
 public class ChimericInstancesCheck extends AbstractQACheck {
 
+    private static final String skiplist = "resources/chimerism_reference_constraint_violations.txt";
```

Comment:
Nit-pick du jour - for consistency, the string "skiplist" could be searched and replaced as "skipList" for variables or "skip list" for comments as it is used as two words in some methods and commonly, as well.